src/client/pfs_new_storage.go

```diff
@@ -128,7 +129,35 @@ func (c APIClient) GetTarConditional(repoName string, commitID string, path stri
 		}
 	}
 	return nil
+}
 
+// ListFileV2 streams FileInfoNewStorage similar to ListFileF
+func (c APIClient) ListFileV2(repoName string, commitID string, path string, f func(fileInfo *pfs.FileInfoNewStorage) error) (retErr error) {
+	defer func() {
+		retErr = grpcutil.ScrubGRPC(retErr)
+	}()
+	ctx, cf := context.WithCancel(c.Ctx())
```

Comment:
This would probably be a good thing to add in the other client side implementations that have code other than the grpc stuff that can error (`GetTarConditional` seems like the obvious one). We don't need it in this PR, I just thought it was noteworthy.

Reply:
Also, just to nitpick while I am here lol, can we name the cancel function `cancel` because that is the name most widely used in the codebase.